### PR TITLE
feat(networks): add support for Base sepolia

### PR DESCRIPTION
**Pull request by clemsos**
# Description

This adds support for Base Sepolia

# Issues

Fixes #
Refs #

# Checklist:

- [ ] 1 PR, 1 purpose: my Pull Request applies to a single purpose
- [ ] I have commented my code, particularly in hard-to-understand areas
- [ ] I have updated the docs to reflect my changes if applicable
- [ ] I have added tests (and stories for frontend components) that prove my fix is effective or that my feature works
- [ ] I have performed a self-review of my own code
- [ ] If my code involves visual changes, I am adding applicable screenshots to this thread

### CODE REVIEW

**Comment by clemsos** [comment]
While creating from frontend locally I am getting the following error

Screenshot 2024-03-27 at 15 11 13

**Comment by clemsos** [comment]
Creating a lock from unlock-app works locally but the UI crash (likely because it is trying to fetch data from staging)

**Comment by julien51** [comment]
> Creating a lock from unlock-app works locally but the UI crash (likely because it is trying to fetch data from staging)

Hum. Maybe from the subgraph?

We should try to get down to the bottom of what exactly fails?

File: packages/networks/src/networks/base-sepolia.ts [hunk 1]
```diff
@@ -0,0 +1,76 @@
...
+    coingecko: 'ethereum',
+    decimals: 18,
+    name: 'ETH',
+    symbol: 'ETH',
+  },
+
+  previousDeploys: [],
+
+  // provider: 'https://rpc.unlock-protocol.com/84532',
+  provider: 'https://sepolia.base.org',
```
Comment (julien51) [inline packages/networks/src/networks/base-sepolia.ts:39]: Ok I'll deploy that shortly!

**Review by julien51 — APPROVED** [review]

File: packages/networks/src/networks/base-sepolia.ts (hunk 1 above)
Comment (julien51) [inline packages/networks/src/networks/base-sepolia.ts:39]:
```suggestion
  provider: 'https://rpc.unlock-protocol.com/84532',
```

**Review by julien51 — CHANGES_REQUESTED** [review]
I run the health check script:

```
[Base Sepolia (84532)]: 0x68F2c5D9009dc4d553f814D689102a53B2b349Cc ❌ Policy below 2!
[Base Sepolia]: ⚠️  ProxyAdmin owner is not the team multisig
[Base Sepolia]: ⚠️  Unlock owner is not the team multisig
```

Please make sure you set it correctly so we don't have any issue later!!

**Comment by julien51** [comment]
I would love to find a way to run these scripts as part of CI when adding new networks!

**Comment by julien51** [comment]
Also I have deployed the provider :)

**Comment by clemsos** [comment]
ok ownership of the contracts have been transferred

**Comment by julien51** [comment]
Thx! Tested and LGTM

**Pull request merged**